styles/speakers.css

```diff
@@ -0,0 +1,61 @@
+.speakers-heading {
+  padding-top: 50px;
+  text-align: center;
+}
+
+.speakers-heading > span {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.speakers-border {
+  height: 2px;
+  width: 60px;
+  display: block;
+  background-color: red;
+  margin: 0 auto;
+}
+
+.featured-speaker {
+  display: flex;
+  margin-right: 40px;
+  margin-bottom: 40px;
+}
+
+.speaker-details {
+  margin-top: 10px;
+}
+
+.speaker-avatar {
+  margin-right: 15px;
+}
+
+.speaker-block-line {
+  border-bottom: 3px solid #ddd;
+  width: 50px;
+  margin-bottom: 20px;
+}
+
+.speaker-name {
+  font-size: 24px;
+  margin-bottom: 10px;
+}
+
+.speaker-role {
+  color: red;
+  margin-bottom: 10px;
+}
+
+.speaker-column {
+  margin-top: 50px;
+  display: grid;
+  grid-gap: 50px 50px;
+  grid-template-columns: 50% 50%;
+}
+
+/* Mobile css - 768px and below */
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] Same here, the CSS styling of this project should be built as a mobile-first.
Please fix it, media queries are supposed to be used in this project to change the design from mobile to desktop, not the other way around.